Userland/Libraries/LibGUI/Calendar.h

```diff
@@ -128,7 +130,17 @@ class Calendar final : public GUI::Frame {
     Gfx::IntSize m_event_size;
     Gfx::IntSize m_month_size[12];
     Mode m_mode { Month };
-    unsigned m_first_day_of_week { 0 }; // Sunday = 0; Saturday = 6
+
+    enum class DayOfWeek {
```

Comment:
This change should be squashed into the previous commit, `Calendar: Support preferred first day of the week`